mmm_experiments/adjudicators/base.py

```diff
@@ -0,0 +1,84 @@
+import logging
+from abc import ABC, abstractmethod
+from collections import deque
+from threading import Lock, Thread
+
+from bluesky_kafka import BlueskyConsumer
+
+from mmm_experiments.devices.switchboard import SwitchBoardBackend
+
+logger = logging.getLogger(name="mmm.adjudicators")
+
+
+class DequeSet:
```

Comment:
Kill the other implementation while doing this move?

Reply:
Definitely. Will do a clean up when I'm confident in the implementation.